pkg/spdk/replica.go

```
@@ -137,6 +138,11 @@ func ServiceReplicaToProtoReplica(r *Replica) *spdkrpc.Replica {
 		res.Snapshots[GetSnapshotNameFromReplicaSnapshotLvolName(r.Name, lvolName)] = ServiceLvolToProtoLvol(r.Name, lvol)
 	}
 
+	if r.BackingImage != nil {
+		backingImageName, _, _ := ExtractBackingImageAndDiskUUID(r.BackingImage.Name)
```

The returned error is not handled.

---

fixed.